sidebars.js

```diff
@@ -561,7 +561,17 @@ module.exports = {
         collapsed: true,
         items: [
           'dev/cli',
-          'dev/cli/espresso-xcuitest',
+            {
+                type: 'category',
+                label: 'Espresso / XCUITest',
+                collapsed: true,
+                items: [
+                    'dev/cli/espresso-xcuitest',
```

Comment:
@spider-sauce The RDC and VDC mockups look great! thanks for putting them together. 

So you think we should split up the NF CLI into three pgs: RDC, VDC, and YAML? 

When we meet this week, I also wanted to ask you about what I have on the main Espresso/XCUITest pages --  moving/deleting some of the content.

Reply:
My thinking for breaking it up was for deep linking + TOC reasons which I can explain tomorrow, but yea we can go over the details tomorrow.